### PR TITLE
tests: remove locker waits

### PR DESCRIPTION
**Pull request by kblok**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -354,3 +354,4 @@ src/Playwright.sln.DotSettings
 src/Playwright/Drivers
 src/Playwright/playwright
 src/Playwright.Tests/Screenshots/**/test.png
+src/Playwright/Microsoft.Playwright.xml
```
Comment (kblok) [inline .gitignore:357]: Is this ok @avodovnik?

**Comment by kblok** [comment]
@pavelfeldman we are getting a consistent fail on `ShouldBeAbleToCaptureAlert`.
It seems that now with the wait message we are sending we introduced a race here

```
var popup = await popupTask;
var dialog = await popup.WaitForEventAsync(PageEvent.Dialog);
```

I'm getting the dialog before being able to wait for the event

``` 
{"guid":"page@c07698ba5ae9a2c4cff3dab3c9b39504","method":"__create__","params":{"type":"Dialog","initializer":{"type":"alert","message":"hello","defaultValue":""},"guid":"dialog@f4690f10a5efec721f53ba98f709a5ff"}}
{"guid":"page@c07698ba5ae9a2c4cff3dab3c9b39504","method":"dialog","params":{"dialog":{"guid":"dialog@f4690f10a5efec721f53ba98f709a5ff"}}}
{"guid":"frame@649140d84a56940fcf64a3420af626d0","method":"loadstate","params":{"add":"domcontentloaded"}}
{"guid":"page@c07698ba5ae9a2c4cff3dab3c9b39504","method":"domcontentloaded","params":{}}
{"guid":"frame@649140d84a56940fcf64a3420af626d0","method":"loadstate","params":{"add":"load"}}
pw:channel:command {"id":73,"guid":"page@c07698ba5ae9a2c4cff3dab3c9b39504","method":"waitForEventInfo","params":{"info":{"apiName":"page.WaitForEventAsync(\u0022Microsoft.Playwright.IDialog\u0022)","waitId":"4a2e1757-d588-46a0-a58f-348aac2076ad","phase":"before"}},"metadata":{"stack":[{"file":"/Users/neo/Documents/Coding/Personal/playwright-sharp/src/Playwright.Tests/PageEventPopupTests.cs","line":89},{"file":"C:\\Dev\\xunit\\xunit\\src\\xunit.execution\\Sdk\\AsyncTestSyncContext.cs","line":58},{"file":"C:\\Dev\\xunit\\xunit\\src\\common\\XunitWorkerThread.cs","line":37}]}}
```

File: .gitignore (hunk 1 above)
Comment (avodovnik) [reply .gitignore:357]: Yeah, this should be fine! I suspect that's why it was set to be in the `bin` folder before?

**Review by pavelfeldman — APPROVED** [review]

**Comment by pavelfeldman** [comment]
> I'm getting the dialog before being able to wait for the event

Re-do the test via popup event

**Pull request merged**